src/common/relayd/relayd.c

```diff
@@ -117,9 +117,63 @@ static int recv_reply(struct lttcomm_relayd_sock *rsock, void *data, size_t size
 error:
 	return ret;
 }
+/*
+ * Starting at 2.11, RELAYD_CREATE_SESSION payload (session_name & hostname)
+ * have no length restriction on the sender side.
+ * Length for both payload is stored in the msg struct. A new dynamic size
+ * payload size is introduced.
+ */
+static int relayd_create_session_2_11(struct lttcomm_relayd_sock *rsock,
+		uint64_t *session_id, char *session_name, char *hostname,
```

Comment:
`*session_id` seems unused? I think its returned by the relayd.

Reply:
Just looked at the code calling this. The session id is indeed returned by the relayd, but it is handled in the caller... The three implementations of this command receive this pointer for nothing.

Remove it from this one and let's remove the other ones after.

Reply:
Done

Reply:
Well the change was done in the partial merge you did from the other pull request #120.